src/gui/src/inspector.cpp

```diff
@@ -992,7 +992,12 @@ void Inspector::update(const Selected& object)
 void Inspector::handleAction(QWidget* action)
 {
   auto callback = actions_[action];
-  auto new_selection = callback();
+  Selected new_selection;
+  try {
+    new_selection = callback();
+  } catch (const std::runtime_error&) {
+    return;
```

Comment:
Should we give the error message to the user in some fashion?

Reply:
Technically the error would appear in the logger.
I suppose here (and in the editor), I could generate a dialog box with the error text.

Reply:
It is logged before being thrown so that should be ok though a dialog would be more obvious.

Reply:
I agree, lets make it a different PR, since the only information in the error is the ID (which is not that useful), so it would require some changes to the logger to throw a "openroad" error (extended from runtime_error) which we can attach the message to as well.